notes/interfaceDef.txt

```diff
@@ -1,17 +1,19 @@
 GLOBAL
-CurrentFloor String(floor_id) Topic - pub by nav
+CurrentFloor String(floor_id) Topic - pub by nav --pubbed
```

Comment:
Don't know what --pubbed means...
Same with the "pubbed" and "subbed" and "called" additions to line 8, 14, and 15

Reply:
Just notes for myself to make sure I implemented everything I needd